br/pkg/checkpoint/checkpoint.go

```diff
@@ -337,6 +379,15 @@ func (r *CheckpointRunner) startCheckpointRunner(ctx context.Context, wg *sync.W
 					errCh <- err
 					return
 				}
+			case _, ok := <-r.lockCh:
+				if !ok {
+					log.Info("stop checkpoint flush worker")
+					return
+				}
+				if err := r.updateLock(ctx); err != nil {
```

Comment:
How about the stability of `updateLock` function, will it introduce some stability issue for backup?

Reply:
The function `updateLock` do 3 things in total:
1. get TS from pd with retry.
2. read lock from external storage.
3. write lock to external storage.

it would fail only when pd or external storage is unavailable to BR.

Reply:
so can we ignore this error. and wait for next time `updateLock` occur?